cloud/store/store.go

```diff
@@ -35,18 +35,20 @@ var (
 	ErrNotOpen        = errors.New("store not open")
 )
 
-type DB interface {
-	AddClass(pl cmd.AddClassRequest) error
-	UpdateClass(req cmd.UpdateClassRequest) error
+// Indexer interface is employed to update both the collection and its indexers in the filesystem.
+// This corresponds to updating metadata for collections
+type Indexer interface {
```

Comment:
Based on the comment above, wouldn't `MetadataHandler` or `MetadataUpdater` be a better name than just `Indexer` ? It seems that it both updates the metadata and the index for collections.

Reply:
This is not related to metadata. I've changed the definition to be clearer